tests/sdk/queries/fileevents/filters/test_exposure_filter.py

```diff
@@ -411,6 +412,44 @@ def test_sync_destination_name_not_in_str_gives_correct_json_representation():
     assert str(_filter) == expected
 
 
+def test_sync_destination_username_exists_str_gives_correct_json_representation():
+    _filter = SyncDestinationUsername.exists()
+    expected = EXISTS.format("syncDestinationUsername")
+    assert str(_filter) == expected
+
+
+def test_sync_destination_username_not_exists_str_gives_correct_json_representation():
+    _filter = SyncDestinationUsername.not_exists()
+    expected = NOT_EXISTS.format("syncDestinationUsername")
+    assert str(_filter) == expected
+
+
+def test_sync_destination_username_eq_str_gives_correct_json_representation():
+    _filter = SyncDestinationUsername.eq("test_user@gmail.com")
```

Comment:
It makes me nervous having real domains (in this case, for email addresses) in tests.  Can we just always use `@example.com`. I know nothing can really go wrong here.

`test_user@gmail.com` is definitely an account out in the wild somewhere

Reply:
Ahh that's a good point. Changed all real domain names to `example.com`